universe/marathon.json.mustache

```diff
@@ -106,11 +106,21 @@
     "network": "HOST",
     {{#service.user}}
         "parameters": [
-        {
-        "key": "user",
-        "value": "{{service.user}}"
-        }
+            {
+            "key": "user",
+            "value": "{{service.user}}"
+            }
+        ],
+    {{/service.user}}
+    {{^service.user}}
+        {{#service.OS_flavor}}
```

Comment:
>  don't quite see the point of using IDs here. Would specifying the nobody user not automatically select 99 on RHEL-based systems? (and 65534 on non-RHEL-based systems)

Yeah the UCR has no problems parsing/translating `nobody` into the appropriate UIDs that match the Host OS but when we run things under the Docker Engine we must explicitly pass the UID (`99` on RHEL/CentOS and `65534` on all the other sane distros) as the `user` param's `value` - It's a known _feature_ :smile:

Reply:
> This field name looks incorrect

Yeah, it _should_ be: [`service.RHEL_family`](https://github.com/mesosphere/spark-build/blob/baafb30364e0c46c70abb593636e7c44425e9d05/universe/config.json#L69) instead of `service.OS_flavor`